pypreprocess/reporting/check_preprocessing.py

```diff
@@ -324,55 +293,28 @@ def plot_segmentation(img, gm_filename, wm_filename=None,
     if cut_coords is None:
         cut_coords = (-10, -28, 17)
 
-    if slicer in ['x', 'y', 'z']:
-        cut_coords = (cut_coords['xyz'.index(slicer)],)
+    if display_mode in ['x', 'y', 'z']:
+        cut_coords = (cut_coords['xyz'.index(display_mode)],)
 
     # plot img
     if hasattr(img, '__len__'):
-        img = compute_mean_3D_image(img)
-    # XXX else i'm assuming a nifti object
```

Comment:
yeah, there're a couple of operations that will become superfluous now with
the new plotting backend.

On Sun, Mar 29, 2015 at 7:29 PM, bthirion <notifications@github.com> wrote:

> In pypreprocess/reporting/check_preprocessing.py
> <https://github.com/neurospin/pypreprocess/pull/80#discussion_r27356606>:
>
> >
> >      # plot img
> >      if hasattr(img, '__len__'):
> > -        img = compute_mean_3D_image(img)
> > -    # XXX else i'm assuming a nifti object
>
> Just curious: we no longer have to compute the mean ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/neurospin/pypreprocess/pull/80/files#r27356606>.
>



-- 
DED